protocol/requests/ban_user_from_community.go

```diff
@@ -0,0 +1,27 @@
+package requests
+
+import (
+	"errors"
+
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+var ErrBanUserFromCommunityInvalidCommunityID = errors.New("ban-user-from-community: invalid community id")
+var ErrBanUserFromCommunityInvalidUser = errors.New("ban-user-from-community: invalid user id")
+
+type BanUserFromCommunity struct {
+	CommunityID types.HexBytes `json:"communityId"`
+	User        types.HexBytes `json:"user"`
+}
+
+func (j *BanUserFromCommunity) Validate() error {
```

Comment:
Here's another funny receiver name. I can't see the pattern, it feels like some kind of puzzle. 🧠